tf_patches/patch.diff

```diff
@@ -32,3 +32,15 @@ index b70ea8af5df..b54a895e306 100644
    } else if (tool_name == "tool_names") {
```

Comment:
General question, how would user config to use `TPU_CAPI` instead of `SE`?

Reply:
For now, we can specify `PJRT_DEVICE=TPU_C_API` instead of `PJRT_DEVICE=TPU` to use the `PjRtCApiClient`. We'll want to flip the default around once the C API is in public libtpu builds.